### PR TITLE
[FIX] Marlin padding

### PR DESCRIPTION
**Pull request by LRL-ModelCloud**

### CODE REVIEW

File: gptqmodel/nn_modules/qlinear/qlinear_marlin.py [hunk 1]
```diff
@@ -157,8 +176,15 @@ def pack(self, linear, scales):
                 self.bias = linear.bias.clone()
 
     def forward(self, A):
+        import torch.nn.functional as F
         A = A.half()
+        if A.size(-1) != self.infeatures and self.infeatures > A.size(-1):
+            A = F.pad(A, (0, self.infeatures - A.size(-1)))
```
Comment (Qubitium) [inline gptqmodel/nn_modules/qlinear/qlinear_marlin.py]: A.size(-1) is called max 3 times if condition matches.

File: gptqmodel/nn_modules/qlinear/qlinear_marlin.py [hunk 2]
```diff
@@ -157,8 +176,15 @@ def pack(self, linear, scales):
                 self.bias = linear.bias.clone()
 
     def forward(self, A):
+        import torch.nn.functional as F
```
Comment (Qubitium) [inline gptqmodel/nn_modules/qlinear/qlinear_marlin.py]: Move this import to top of file.
